### DISCUSSION TITLE
border-l-1 is not working

### DISCUSSION THREAD

**Discussion by pratyushtewari** [tailwindlabs/tailwindcss#8006] [Help]
I want a left border of 1px but it is not working. In fact all odd number borders are not working.
`border-l-1 border-l-3 ...`

can anyone suggest some workaround?

**Comment by gungim** [comment]
you can add config for border in your tailwind.config.js
<img width="366" alt="Screen Shot 2022-04-01 at 11 36 10" src="https://user-images.githubusercontent.com/50023960/161195686-e7093e14-ad2e-40fb-9238-2bdb8df16edf.png">

**Reply by NovusOpus** [reply]
`border`, `border-l`, `border-r`, `border-t` and `border-b` already do that, so you don't need to extend a new value for that 1px border, unless for some very specific scenario.

Otherwise if you are pointing it as an example of how to add/extend custom values in your `tailwind.config.js` and you already knew that `border` or `border-*`  = 1px border, then my apologies, I'm just bringing it up in case you did not know.

**Reply by toocomputer** [reply]
Just venting here, but really confused and frustrated about `border-l`, `border-r`, `border-t`, and `border-b` not being mentioned anywhere in the Tailwind documentation. Maybe I'm missing something?

**Reply by NovusOpus** [reply]
@bozoputer Yep, you're missing it: https://tailwindcss.com/docs/border-width, you can see all the border classes with their respective values in the list at the start.
Also with the examples in this section: https://tailwindcss.com/docs/border-width#individual-sides, more specifically: `border-{side}` , `border-{side}-0`, `border-{side}-2`, `border-{side}-4`. They did not add an example of every single border class since that would be redundant.

**Reply by toocomputer** [reply]
Ahhh, I figured it was something I was overlooking (it's not the first time it's happened on their site where I've missed all of the text in that scrolling window 🤦🏽‍♂️). 

Thanks so much for pointing it out! 🥂

**Comment by NovusOpus** [comment]
Just use: `border-l` for a 1px left border, as for any other odd or custom value you can use [Arbitrary Values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values), for example: 
`border-l-[3px]` `border-l-[5px]` `border-l-[3em]` `border-l-[3rem]`
